lib/Gocdb_Services/Config.php

```diff
@@ -567,4 +567,59 @@ public function getEmailTo()
 
         return $emailTo;
     }
+
+    public function getIdentityProvidersInfo(): array
+    {
+        $localInfo = $this->GetLocalInfoXML();
+        $identityProviders = [];
+
+        if (!empty($localInfo->identity_providers->provider)) {
+            foreach (
+                $localInfo
+                    ->identity_providers
+                    ->provider as $providerDetails
+            ) {
+                /** idp */
+                $idp = (string) $providerDetails->idp;
+
+                /** name */
+                $name = (string) $providerDetails->name;
+
+                /** authentication_realms */
+                $authenticationRealms = [];
+                if ($providerDetails->authentication_realms) {
```

Comment:
Given https://github.com/GOCDB/gocdb/pull/524/files#r1842006878, this will need a slight tweak

Reply:
DOne. 5091398904b06339ae9493205330b88a6b8faac2